### PR TITLE
Ability to join event and keep track of participants

### PR DESCRIPTION
**Pull request by MorrowCameron**

### CODE REVIEW

File: Eat_and_Greet/src/comps/NumberInput/index.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  right ?: any;
+  width ?: any;
+  height ?: any;
+  onTextChange: any;
+}
+
+const NumberInput = ( {placeholder, top, left, bottom, right, width, height, onTextChange} : NumberInputProps ) => {
+const [number, setNumber] = useState("");
+
+const onChanged = (userInput : string) => {
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/comps/NumberInput/index.tsx:19]: Makes Decimals impossible, no spot cap.

File: Eat_and_Greet/src/comps/RoundedButton/RoundedButton.tsx [hunk 2]
```diff
@@ -31,11 +31,9 @@ const RoundedButton = (props : RoundedButtonProps) => {
     })
 
     return (
-            <View style={[styles.CornerButtonOutline, localstyle.placement, localstyle.sizing]}>
-               <TouchableOpacity onPress={props.onPress} disabled={props.disabled}>
+        <TouchableOpacity onPress={props.onPress} disabled={props.disabled} style={[styles.CornerButtonOutline, localstyle.placement, localstyle.sizing]}>
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/comps/RoundedButton/RoundedButton.tsx:34]: Actually clicks button not text now.

File: Eat_and_Greet/src/screens/PostEvent/index.tsx [hunk 3]
```diff
@@ -1,22 +1,23 @@
 import React, { useState } from "react";
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/screens/PostEvent/index.tsx:1]: Organized imports a bit

File: Eat_and_Greet/src/screens/PostEvent/index.tsx [hunk 4]
```diff
@@ -68,21 +91,23 @@ export default function PostEvent() {
           Alert.alert("This Event is Already Happening Today");
         } else {
           const numberOfEventsSnapshot = await getDocs(collection(db, "Number of Events"));
-          let uidList : number[] = await getNumberOfEventsData(numberOfEventsSnapshot);
-          let uid : number = uidList[0] + 1
+          let uid : number = await getNumberOfEventsData(numberOfEventsSnapshot) + 1;
+          const host = name;
```
Comment (MorrowCameron) [inline Eat_and_Greet/src/screens/PostEvent/index.tsx:95]: Could be unnecessary variable creation

**Review by Logidoke — APPROVED** [review]
Looks like you got an angel room to me

**Pull request merged**